code/headers/usart_connection.hpp

```diff
@@ -1,16 +1,15 @@
 /// @file
-/// @brief      base class for a usart implementation hardware, software and mock
+/// @brief      base class for a usart implementation hardware, software and
+/// mock
```

Comment:
is test now

Reply:
On this moment on the release this is what the comments are.